### ISSUE TITLE
When using example get must be compatible with PhpCsFixer\AbstractFixer::applyFix(SplFileInfo $file, PhpCsFixer\Tokenizer\Tokens $tokens): void in /home/josh/Code/AWIN-Feeder-Wordpress-Plugin/vendor/tareq1988/wp-php-cs-fixer/src/SpaceInsideParenthesisFixer.php

### ISSUE THREAD

**Issue by OdinsHat** [tareq1988/wp-php-cs-fixer#1]
When running this at the command line:

`vendor/bin/php-cs-fixer fix`

```php
PHP Fatal error:  Declaration of WeDevs\Fixer\SpaceInsideParenthesisFixer::applyFix(SplFileInfo $file, PhpCsFixer\Tokenizer\Tokens $tokens) must be compatible with PhpCsFixer\AbstractFixer::applyFix(SplFileInfo $file, PhpCsFixer\Tokenizer\Tokens $tokens): void in /home/josh/Code/AWIN-Feeder-Wordpress-Plugin/vendor/tareq1988/wp-php-cs-fixer/src/SpaceInsideParenthesisFixer.php on line 63
Script vendor/bin/php-cs-fixer fix handling the fix event returned with error code 255
```

**Comment by SirLouen** [comment]
Same here

This project seems to be abandoned :(

**Comment by OdinsHat** [comment]
@SirLouen I may fork it and update with bug fixes if I get time.

**Comment by tareq1988** [comment]
Fixed in e9eae514c24d007c9f9ea868856653c2c88f880e
